packages/legend-extension-external-language-morphir/src/MorphirGenerationExtension.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import packageJson from '../package.json';
+import { AbstractPreset } from '@finos/legend-shared';
+import type { StudioPluginManager } from '@finos/legend-studio';
+import { MorphirGenerationPlugin } from './MorphirGenerationPlugin';
+
+export class MorphirGenerationPreset extends AbstractPreset {
```

Comment:
Please follow the convension to prefix your extension with something like `EL_Morphir...` as `EL` stands for `ExternalLanguage`

Reply:
Fixed! Renamed with ELMorphir_ prefix.